ttt-game/src/components/board.js

```diff
@@ -0,0 +1,102 @@
+import React, {useReducer, useEffect, useContext} from 'react'
+import PlayerOne from './player-one.js'
+import PlayerTwo from './player-two.js'
+import TimeMachine from './TimeMachine.js'
+import { ScoreContext } from '../contexts/ScoreContext.js'
+
+const board = [null,null,null,null,null,null,null,null,null]
+let turn = 0
+let timeSteps = []
```

Comment:
To respond to your global variable comment - I'd probably just nest these inside of the component :-) Another option would be to create a custom hook called `useBoard` or something that could hold these initial values, plus the `useReducer` call below.